<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatRoom</class>
 <widget class="QDialog" name="chatRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chat room</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>0</y>
     <width>331</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextBrowser" name="browser"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="emitButton">
        <property name="text">
         <string>&amp;Emit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>&amp;Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>emitButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>browser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
